steps/android-info/1.0.1/step.yml

```diff
@@ -0,0 +1,51 @@
+title: Android info
```

Comment:
`title` should be `APK info` instead of `Android info`, because `APK info` would better explain, that this step will provides infos about apk file/files.